src/network/HTTP.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+export default class HTTP {
+  constructor({ protocol = 'http', hostname = 'localhost', port = 8181, token }) {
```

Comment:
We should still support passing `pathname` parameter.

Reply:
Done!